.github/workflows/envoy-verify.yml

```diff
@@ -0,0 +1,87 @@
+name: Verify/examples
+
+on:
+  # This runs untrusted code, do not expose secrets in the verify job
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: "Git SHA ref to checkout"
+      sha:
+        description: "Git SHA of commit HEAD (ie last commit of PR)"
+      head_ref:
+        description: "Ref for grouping PRs"
+
+concurrency:
+  group: ${{ github.event.inputs.head_ref || github.run_id }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  check:
+    if: |
+      ${{
+          github.repository == 'phlax/envoy'
+          && (!contains(github.actor, '[bot]')
+              || github.actor == trigger-workflow-verify-examples[bot])
```

Comment:
need update.

Reply:
gonna give it one more wip run before i fix these